pysteps/blending/steps.py

```diff
@@ -809,23 +825,80 @@ def forecast(
             if measure_time:
                 starttime = time.time()
 
+            if precip_models_cascade is not None:
```

Comment:
I dont think precip_models_cascade is ever initialized, al I wrong in this?

Reply:
On line 576 it is set to None and then on line 579 it is set to the input `precip_models` if the ndim is not equal to 4.

Reply:
So that means if dims is not 4 (the already decomposed case), it has been defined. If it is the other case (raw NWP), it is non, and thus, this decomposed. Is this correct?

Reply:
Yes that is correct.